sig-scheduling/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG Scheduling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+We build Kubernetes schedulers and scheduling features for Pods. We design and
+implement features that allows users to customize placement of Pods on the nodes
+of a cluster. These features include those that improve reliability of workloads,
+more efficient use of cluster resources, and/or enforces placement policies. 
+
+### In scope
+
+Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Scheduling related features (e.g. Node Affinity)
```

Comment:
Suppose ML/Bigdata workload supportings are in this catalog :)

Reply:
Yes, the scheduling related requirements of running ML or Bigdata workloads fall into this category.